testing/solidity/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Trevis Schiffer <nikolaibitinit@gmail.com>
+# Maintainer: Trevis Schiffer <nikolaibitinit@gmail.com>
+pkgname=solidity
+pkgver=0.5.10
+pkgrel=0
+pkgdesc="The Solidity Contract-Oriented Programming Language"
+url="https://github.com/ethereum/solidity"
+arch="all"
+license="GPL-3.0-or-later"
+depends="libgcc"
+makedepends="boost-dev boost cmake git"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/ethereum/$pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
```

Comment:
Default value, can be removed

Reply:
This one?  builddir="$srcdir/$pkgname-$pkgver"

Reply:
Yes, that's the default value, so you can remove the builddir.